components/FormFieldSelect/FormFieldSelect.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FunctionComponent } from 'react';
+import { Select } from '../Select';
+
+interface FormFieldSelectItem {
+  value: string;
+  label: string;
+}
+
+interface FormFieldSelectProps {
+  label: string;
+  options: FormFieldSelectItem[];
+  themeColor: 'light' | 'dark';
+}
+
+const FormFieldSelect: FunctionComponent<FormFieldSelectProps> = ({ label, options, themeColor }) => {
+  return (
+    <>
+      <label htmlFor="topic" className="form-label">
```

Comment:
Only show if there's a `label` prop. Also the `htmlFor` has to be dynamic, or ditch it entirely. May not be useful here.

Reply:
Added conditional rendering and ditched the `htmlFor`